reliability-engineering/terraform/modules/github-actions-connectors/zendesk-estate.tf

```diff
@@ -0,0 +1,61 @@
+resource "aws_iam_openid_connect_provider" "github_actions" {
+  url = "https://token.actions.githubusercontent.com"
+
+  client_id_list = [
+    "sts.amazonaws.com",
+  ]
+
+  thumbprint_list = [
+    "a031c46782e6e6c662c2c87c76da9aa62ccabd8e"
```

Comment:
Where does this magic string come from?

Reply:
it is just a magic string! https://stackoverflow.com/a/69247499

I can put this as a comment if you want :)